### ISSUE TITLE
Niantic changed API

### ISSUE THREAD

**Issue by OlesZadorozhnyy** [mchristopher/PokemonGo-DesktopMap#621]
Niantic changed API an hour ago. That is why application doesn't work

**Comment by pocketq10** [comment]
it was only a matter of time...sigh lol now we are all really left blinded

**Comment by IamMetuZ** [comment]
Source?

**Comment by mangantj** [comment]
@pocketq10 Does not mean this app will not work. Just a change to the api.

**Comment by kringon** [comment]
They most likely found a way to detect non-clients and instead of blocking their requests, they are sending empty responses

**Comment by mg6101** [comment]
https://github.com/mchristopher/PokemonGo-DesktopMap/issues/608

This is how you change the scan timing to change the default scan time of once every 5 seconds to once every 10 seconds. Anything faster than 10 seconds will not show Pokemon. Either shrink your radius to compensate or run multiple instances with multiple PTC logins at appropriate time offsets.

**Comment by pocketq10** [comment]
@mangantj  app will start work but like kringon said empty responses

**Comment by textcursor** [comment]
damn not again

**Comment by mg6101** [comment]
Damn. Looks like they changed quite a bit on the API side. All scanning is broken. :(

**Comment by EternisRequiem** [comment]
@mg6101 So you can't fix it by scan timing, huh? ;)

**Comment by thatoneguyyouallknowbutdontlike** [comment]
@EternisRequiem nope. changed the scan time in config.ini, and no dice. Until we figure out the API changes the app will be down.

**Comment by EternisRequiem** [comment]
For information for you dev-type-people that can actually make the things:
[Thank you Pokemongodev Reddit](https://www.reddit.com/r/pokemongodev/comments/4w0jum/all_ptcgoogle_logins_failing_from_api/)

**Comment by obamaphone** [comment]
The switching to 5-10 seconds method does not work anymore :(

**Comment by psychoblue** [comment]
This sucks.  I'm not spending another cent on this crap game.

**Comment by obamaphone** [comment]
It's really sad that Niantic chose to go down this path. They'll find out the hard way when people stop playing and buying stuff from the store because they're bored out of their mind from catching Pidgy and Capterpie all day.  

Instead of being Pokécops, why not incorporate a usable tracker **that actually works?**  I wouldn't mind a little sonar circle that shows up on the map to show you the general direction a Pokemon is. They don't even have to show you it's name, just color code the common ones with a green ring and the rare ones with a red ring.

**Comment by ShiroTora** [comment]
so this error i get is bcuz it dont work anymore ? 

http://imgur.com/a/AecZS

**Comment by majuss** [comment]
@ShiroTora it seems like a different error. But the app wouldn't work even if it launches so no big deal. I will also quit if they are going to block every scanner...

**Comment by joshuacane** [comment]
@majuss  Same. My friends and I are no longer playing the game until the scanners come back online, and if they never do, then we have to sadly let go of Pokémon GO. This is what happens when a company other than Nintendo gets to make a Pokémon game...

**Comment by ShiroTora** [comment]
as far as i readed , they have a dynamic api, that change when they detect a scanner, OR its a encrypted signal witch make the prog/app  sends empty data instead

**Comment by Shogunz1** [comment]
You know, PG is being released in Brazil/Mexico right now...where the Olympics are getting ready to start. Could be another system-wide fart.

**Comment by asavage44** [comment]
Changing the API might not be just to block 3rd party applications. There are many legitimate reasons to update the API. We'll see once the community has figured it out.

**Comment by Shogunz1** [comment]
Everyone just needs to calm themselves.. this time tomorrow there will undoubtedly be a fix for this and all will be well in the world again.

**Comment by Gentleman-Taco** [comment]
Thanks Obama!

**Comment by Shogunz1** [comment]
Lol

**Comment by tinarazzburton** [comment]
Is there a proper forum for discussing this app or is this it?  This had been the best pokescanning app I tried, by far.  I'm curious if there is another that is more popular than this.

**Comment by kringon** [comment]
Looks like they are working on a fix for the sources: https://github.com/NECROBOTIO/NecroBot/commit/35339c3f527e7f934ae5fae6a7c7aa45e246be32

**Comment by joshuacane** [comment]
I can't code for the life of me, so I just wanted to say thanks to all of the people who are working endlessly to get this app functional again. I've tried many other apps, but they include opening up a terminal(don't know what that is) and copying and pasting code, and getting a program to make your computer a server(???)... Then more code to paste into places that I don't know(??????). It's sufficient to say that this app is mine and many others only hope.

**Comment by Shogunz1** [comment]
I'm definitely donating if it gets back up.

**Comment by majuss** [comment]
@joshuacane just type terminal into the sportlight if you are under os x. Every scanner software makes your computer a server... running scanners from terminal is faster, saves ressources and is very easy to automate with multiple accounts. So the reality is, there is no real reason for a GUI app.

**Comment by HuXu77** [comment]
Fixes are going up, just need to integrate with this app https://github.com/PocketMobsters/PokeMobBot/commit/eb1c6abff719db20d652e32f781931c7a6b0f544

**Comment by mosmi** [comment]
Amazing. That was so quick!

**Comment by psychoblue** [comment]
Once this installer gets updated/fixed to 0.1.7, I'm making a donation.

**Comment by KikkomanChow** [comment]
Probably not a popular take on this. A greater part of me hopes that this does not get fixed. 
Just have to wait for them to introduce a new in game tracker, and get back to a level playing field.

**Comment by JSchwerberg** [comment]
@HuXu77 How is integrating with a commit that didn't fix the problem going to fix the problem?

**Comment by Shogunz1** [comment]
Me too. I'm a disabled vet and this program has made it so I can actual play this game. I love it and will support it.

**Comment by sushab** [comment]
the game isn't fun without this app. I've just spent 20 mins outside looking for a Dratinin and didnt find it, regardless of knowing the EXACT 2-3 spawn points just close by home (thx to this app). The Ingame radar had me fixed to the closest mon and I still couldn't find it. 

Also without this app playing above lvl 20 is pretty boring. If U want to step up your game and find rarer creatures, walking casually (daily) isn't going to cut it and the radar just doesn't help any, as just mentioned. This kinda app gets you hooked, on knowing rare spawns around your house and guessing if you can get there on the close windows thats spawns provide is kinda thrilling. 
The alternative, playing 'casually' and hoping in good finds as u go home on the bus isn't viable once you've lvled to lvl<20s (casual or hardcore) because you've already done a lot of it. Rarer finds are basically the only dragging force in the game, considering the futility of gyms. Frustrating every effort to improve ones chances of doing so, only suggests that Niantec is worried about the community burning content too fast. The easy fix would be to increase the xp curve but as its already been pointed out (by hc players) that the curve is already rather steep, what they have left is slow down the curve at which players burn content. 

in short. I don't see the ingame radar (even in its previous form) being anything but remotely usefull. ty pokemon go online map!

**Comment by pill0ws** [comment]
well, it would be great if they followed this shit up with an actual improvement to the game. They have fought tooth and claw to prevent Map scanning While GPS spoofing was still possible. Have they stopped both? or are they more concerned with us knowing things than us 'cheating'? I jyust want to play the fucking game normally, they seem to want it to be absolutely impossible to play at all

**Comment by SquireHaligast** [comment]
So will things just work once a fix is found, or will we need to perform repairs manually once we know what to do? Can tech-savvy people let us know whenever possible what needs to be done? 

As you can tell--I am not one of those people! :)

**Comment by sushab** [comment]
The content issue is pressing I  insist,
considering that GYMS:
A) are rather useless in terms of rewards, with 5 gyms u get 1000 dust and 50 credits. Considering the effort that takes (unless using a car.. and consider the competition for those who don't) to keep 5 gyms long enough to collect, (both in a city or in .. Yukon Ho) its hard to expect people to invest so much 'augmented reality time' (and gasoline or calories or.. whatever) for such little reward. Which brings to point B
B) Some people will compete at such levels. Making it near impossible for children to take GYMS. Now if you're on these forums your likely not a child. And play Pokemon GO. How many children have you seen playing pokemon go recently? I saw several. lets leave it at that. Another question have you ever played a child oriented videogame? like Disney Infinity? No? Just saying you can't put children at the same level of competition with adults. EXPECIALLY gamers.. The time gamers are willing to invest .. is a lot. GYMS are just .. a lot of trouble for Niantec, gamers want them more competitive/significant while children surely don't want to be bullied or even lose when playing videogames.. Hell, who wants to?

Without Gyms, we're back to rare pokemons. The game needs new features. Niantec is wasting time doing whatever ISNT working on new game features.. PERIOD.

**Comment by bukow10** [comment]
what's the status on this api workaround?

**Comment by pokehunter24** [comment]
If Niantec spent there time and money in to Scaling a rollout of the game they wouldn't need to shut down tracking websites even they agree it makes the game better but there servers cant handle the players and the trackers so they kill the trackers, they are seriously fire fighting and by doing so they are making bad judgment calls.

P.S I need this tracker working for just a bit longer I'm 5 pokemon off of the total 153

**Comment by sushab** [comment]
and I just have to add...

NOT KNOWING exactly where a 'mob'... sorry, mon, is, is DANGEROUS! Well yes, with this APP i can fetch that Dratini with precision, while the in game RADAR may even have me cross private property or... perillious.. (sigh) terrain', while attempting to get closer to said. mob... mon. Not knowing exactly where to go, in a city is NOT good, especially for children. 

Niantecs stubbornness on the issue is ... baudaciously bedazzling ... and encourages the dangerous play they tell players to refrain from. Considering children are playing augmented reality I think they should be using their lawyers more wisely....

**Comment by EasyPeasyEevee** [comment]
Thanks to everyone working on this. Loved this while it lasted. Sigh.

**Comment by blonkpuff** [comment]
Give them a bit to try to get this working peeps.  Keep it up guys!

**Comment by Regulator69** [comment]
im pretty convinced they will figure it out from what i see, but no matter what happens, you guys are the BeezNeez! if only niantic had these guys, they would put trumps earnings to shit with a good support team.. but dreams dont come true most of the time lol

**Comment by pokehunter24** [comment]
Pokemon Go Wishlist:
1. Pokemon Sonar showing which ones are near with there outlines.
2 Egg incubator percentage on the map screen.
3. Pokemon trading capabilities so we can swap with others.

wouldn't that make it better :)

**Comment by sushab** [comment]
no not really.

Observing the map for a couple days, noticing a pattern, spotting a rare spawn, respotting it, estimating its spawn time, realizing you have a rare (once or twice a day spawn) close to home, competing with a couple of other local gamers to catch it.. Well, eventually catching it, is.. cool.. exciting? requires some effort? gives u back quite a reward/thrill when u finally have that mon in a bag. Sort of .. could be, compared to winning the stranglethorn fishing extravagantia in the good old days? certainly deserves a Yatta! maybe just a whisper if its after midnight?

Expecting gamers not to have this info.. is .. just silly, brakes with gaming 'protocol', is .. boring, and just stinks of pain in the ass just cos, and tell u what i'll add a sparkle of passive agressive and tweet you once a week, if I feel like it.... -.-

**Comment by frankenbilly** [comment]
I added another Pikachu to my inventory because of this app today. It's sucks that we can't use these tools.

**Comment by Regulator69** [comment]
yep i got another snorlax today and a lickitung before it went down, then i had a pikachu on my nearby even in the first spot, so i went to all the spawn spots around me that i've seen and to no avail... but like ive seen that rare despawn on the map, and that pokemon will stay on your nearby sometimes for a few minutes.. so sometimes you just search for a pokemon thats already despawned, and thats fun and an effective system huh? lol k.. whatevs...

**Comment by lazyTrainer** [comment]
Grumble. Grumble. Good luck getting this working again. As stated before by others. I'm out till its back. I'll check back in a few days. Thanks for the hard work.

**Comment by pancakebatter12345** [comment]
Ive noticed that hovering over lured pokestops says something like "Lured pkemon: pidgey" etc., so it still seems to be picking those up. As far as what ive seen on my app, the lured pokemon seem to be accurate. If the api change was deliberately done to stunt 3rd party trackers, they certainly missed some things.

**Comment by pill0ws** [comment]
Pokemon go Wishlist
1. Scan radius equal to effective range of the tracker
2. removal of the tracker outright
3. Pokemon specific sounds when they spawn so I can ignore the rats and birds and enjoy nature until I come across something cool

These map scanners rose out of necessity. If they removed the need to struggle so hard to actually find things then there would be no reason to use them. Anyone whose used the apps knows that often theres a rare in someones backyard and you have to creep up real slowly to the house and very close to trigger it. Thats a big problem for this game as it practically encourages tresspassing and using the tracker youd be tresspassing in vain, finding nothing at all but knowing theres a damned Charmander nearby. The rares are often spread out, 70 meters at 10s updates encourages tresspassing even more and pretty much also encourages loitering and looking even more suspicious because you are trying to scan areas at the the circles border.

I'd love if there were no reason to use this kind of software at all, unfortunately the game was unplayable without it and even less playable at 10s per 70 meters (as well as whatever other security measures they put under the hood, I suspect jumping locations triggers something too. after I capture something If I make it too far down the road the avatar will jump locations and I tend to not find anything for a bit)

**Comment by bmeucci** [comment]
1. If it's set up for detecting mapping apps it will look for extremely regular distances between scan requests (machine-like behavior) so randomizing that will help. 
2. It will probably also limit the speed between scan locations to 55-60 MPH so that will be another way to look legit.  (IE If the distance between scan locations / time between scans > 60 MPH they will likely kick it out)
2b. Duplicate accounts in vastly different locations will be one of the first easy "tells" so that's a no-no. (and also conforms to the speed restriction rules they would put in place.) So anyone who uses the app must also use a dummy account, not their own.
3. If there are over 50-100 accounts making requests from the same IP address that would be another marker of a mapper app. You're going to have to spoof IPs or something.

4. The final most annoying/difficult countermeasure: You'll need to make every dummy account attempt to grow its pokemon collection though occasionally hitting pokestops and attempting to capture monsters. This seems too large to handle programmatically and may be something you'll have to ask the users to maintain.

Those are what I'd put in to stop you. If anyone else has some ideas, lets put them out there. Solve all these "tells" and any countermeasures they put in specifically to block you should be circumvented. IE: you should be organic enough to look indistinguishable from a normal player.

**Comment by EternisRequiem** [comment]
[Did the API thing get fixed and can be implemented for this now too?](https://github.com/PocketMobsters/PokeMobBot/pull/559/commits/e27df43f373c4af4aee80c0c2cdcac42e88ac4e6)

**Comment by blonkpuff** [comment]
^naw to a lot of this dude

1.) Niantic isn't going to scan for "extremely regular distances" between scan requests, especially since a lot of people go the same routes every day because that's their established pokemon stop route etc.  If a human can theoretically do it it's going to be allowed.

2.) No to this too, they don't even soft ban regular account moving this fast because highway passengers in cars have pokemon go on all the time, even on bullet trains yo. They only prevent scans after a huge distance warp and even that's only a soft ban that lasts 30seconds-1hour which wouldn't break this app.

3.) Maybe, but most people are using this app in one instance from a computer and the other they log in later on their phone which inherits a new ip from their cell carrier. Also if you were a programmer you'd know that asking for a hackeresque system to spoof ip's just for pokemon in an application like this is a bit of a crazy request.

4.) Completely no.  All my wat.  Niantic isn't going to ban accounts that sit idle unless maybe a year passes and they are tidying their database and even then probably not. Banning accounts that don't "grow their pokemon collection"  would be an insane enactment that would do little to stop mapping and much more to punish people who installed the game and simply haven't gotten around to hunting. 

I get what you're saying about organic but the tells are described in the api, which are not *that* restrictive and which these guys have and can decipher well.

**Comment by bmeucci** [comment]
@blonkpuff 

Unless you're a developer I'm really not interested in discussing it with you. You don't seem like one since you just ignored a shit ton of obvious stuff.

1) Humans don't move perfect distances in perfect times regularly
2) They aren't banning they are skipping supplying data which makes it fine for regular users
3) I am a programmer, but a bit of a dinosaur admittedly. But without looking, based upon the number of proxy sites readily available I'd be willing to bet the code to accomplish it is readily accessible
4) Wow, how on earth are you missing such obvious stuff? My god, I hope you're not a programmer. If an account moves around logged in and never tosses a single ball, accesses inventory etc etc for a month, that's a fucking bot, period.
...then again I guess there are code monkeys out there that never do a lick of design... still though, I can't think even a code monkey could miss this.

I'm not interested in arguing further so if you try to continue this line of argument you will get the last word to nobody caring. I'm providing useful ideas and you're failing to shoot them down with anything even remotely broadly considerate..

**Comment by cykou** [comment]
can we just fix pokemon desktop please ;/

**Comment by obamaphone** [comment]
I stand with Niantic with this one: Kill the Poketrackers and incorporate a **KICK ASS TRACKING SYSTEM** into the app!

 Wait... They did say they were going to add... Um, update...???

**Comment by blonkpuff** [comment]
bmeucci, 

1.) You clearly missed the point.  It doesn't matter if humans don't move that way naturally.  They *can*, it's possible, and if niantic banned that they'd be insane.  How do you not comprehend that?
2.) You're just swapping a term.  They don't ban *or* skip supplying data to people moving that fast. That's common knowledge.
3.) You're definitely a dinosaur if you think writing spoofs/proxies into an application like this is a good idea.
4.) You missed the point again. Thank god you're not a developer because if you went to discuss the terms of autobans with fellow developers and said "Hey guys let's ban anyone who doesn't throw a ball after a few months and doesn't access invetory because it's clearly a bot" they would LAUGH YOU OUT OF THE BUILDING.

With galactically uninformed statements like yours (getting all hostile over someone pointing that out)...I'll certainly agree that arguing with you is pointless.

**Comment by pill0ws** [comment]
1) The only thing I see as a possible "tell" that a login is an app is the reset to a specifically accurte coordinate every cycle. Common routes surely arent a good method to determine a bot, but f every interval of X the login requests the precisely same set of scan queries, that seems to be a little bit of a different story

**Comment by bmeucci** [comment]
Okay bud, keep pretending. Love them posers... So hilariously not getting it and just committing harder to their nonsense as though BS confidence will actually sway people who understand.... Management material right there. Mike Judge would be proud and I know higher-level management would buy into it too. Seeing people like that flood in to startups riding on top the accomplishments of the actual programmers is what got me out of the business. A decade of seeing that repeated over and over was more than enough.

"I think we need to synergize our action items and table the discussion. We need a more agile analysis of our customer facing experience. I think we can just pivot backend resources and bring in short-term out-of-house talent for a rework of critical systems and the in-house resources can integrate it in their down-time while the server-farm hides the transition!"

So anyhow, thanks for the actual work guys and I hope the strategies I mentioned help you anticipate their attempts to stymie you.

**Comment by blonkpuff** [comment]
@bmeucci 

Straight up, I think you're embarrassed that your naive pseudologic got called out. :u 
It's cool that you want to contribute but I'm not a dick for calling attention to how uninformed your ideas on the subject are. I'm confident the devs here know better but it's still misinformation coming from a self proclaimed out-of-the-loop programmer, whereas I am very much not. I don't like confident misinformation because people swallow it up. 

@pill0ws 

I hate to keep on it but they aren't like to ban for that either. I'm not saying some of the theories put forth by you and bmeucci wouldn't catch bots, they totally would, but that's not the kind of thing programmers often design bans for in geo games like this.  You have to think like you're in their board room. For every stationary coordinate ban that went down they'd get one bot and 10,000 (lazy but regular) users who log in from their seat every day because their seat is within range of a poke-stop.  Auto bans when micro transactions are at stake are a warily considered thing in this type of mobile development because even the most blatant hackers can potentially put money into the game.  There are what...9.5 million active users daily on PokeGo? They already shut down pokevision (who were asked, not api blocked.) So now they only have to worry about a few thousand people technically inclined enough to go get a google maps api key.  No it's not hard, but it's enough to dissuade a lot of people who simply used poke vision.

Point is, the moment Niantic puts something in that's going to catch some hackers but could potentially outst thousands of legitimate players they've got problems. They aren't going to do something that can threaten an income stream on that level to battle an app like this with its limited user base. I've done facebook app development almost since its inception. Facebook changes their api like a new dad changes diapers and it breaks apps almost every time. That's what's happening here, the devs just need time to reconfigure. 

Just an FYI, in this industry Autobans are done less by user behavior and way more by things like a third party application's digital signature/imprint/metadata. They will always ban application clues like that over local profile behavior. I'm not saying some behavioral bans won't happen, but when they do implement those they are the type that are all the way obvious, like warping from New York to California in 30 minutes. And again since money is at stake, even that's only a hour ban.

**Comment by pill0ws** [comment]
Ive been suggesting an alternate pattern to scanning for some time now (I suck at programming, I know enough to get a B in an introductory college class about 5 years ago, thats about it, otherwise I'd of tried doing this myself already). Mainly because the spawn points are all static, after scanning the area so many times you notice that pokemon are always in nearly the exact same spots .  More importantly theres just areas that literally never have any at all. So scanning these areas, especially at the 10s interval, is a complete waste of resources. Whether they are triggered by a repeated pattern or not, the pattern itself is wasteful of time now that update rates are set to 10s

**Comment by blonkpuff** [comment]
@pill0ws 

That'd be wild but their would need to be one hell of an algorithm going on in Poke Go Live Map that detects and then registers all those static spawn points all over the world.  If they aren't put forth via the api it's going to have to be processed client side which is going to be craaazy.  I think your idea would be best done in a few chosen points like central park when everyone seems to be spoofing.  That way (if the points actually are static) the app could come with a preknown list of coordinates to scan in that area.  But again, that might leave an application footprint like I was saying and having a log of all the spawnpoints everywhere is a tall order.  Cool thinking tho~

**Comment by RvdHNL** [comment]
From what i read on reddit, it will be back to 5secs. They seem to be using a hash for encrypting the pokemon. Some apps still can be used to scan your inventory and stuff, so only mapping issues seem to remain.

https://www.reddit.com/r/pokemongodev/comments/4w0jum/all_ptcgoogle_logins_failing_from_api/d634g5s

**Comment by pill0ws** [comment]
@blonkpuff 
I mentioned being a complete noob in programming but I presonally dont need to scan all over the world. Couldnt there just be a log of every discovered pokemons coordinates, time/date that populated from use of the app? You'd store all that kind of stuff client side anyways. 

Additionally: I am curious is the only data being sent back from these scans is the location and despawn timer of pokemon. There are some other variables I have noticed simply playing the game. Orientation being one of them, which direction is the pokemon facing? I believe they respawn later in that direction, before they killed the maps, I had been able to predict when and where (using a decent sized radius) that the next Snorlax spawn would occur. He is so common in my area that patterns started to emerge from logging his appearences. Despawn timers often shared an offset of 30min, he spawned for 15 at the time and the despawn timeres for the ones I "tracked or followed" were always in 30min chunks. So I tehorized taht they despawn for the same duration as they spawn and they respawn in the direction they were oriented toward when spawned. This falls in line with how I have succesfully made use of the in game tracker. Imagine that there are tracks for these invisible critters. Imagine that the tracker points in the direction of those tracks relative to your avatars orientation when you cross them. I've had fairly regular success with this approach but some inconsistencies still occur. However, just assume this is all true for one second, where would those tracks be coming from? I was literally just about to go out and test all of this the night Niantic swung their dongers at the the map scanners, so I cant say it is certain. however the data i collected from the app suggests that there isnt magically 20+ Snorlax appearing in my area every day, that they appear either the west, east or northeast edges of town and wehn they disapperar, appear again in those general directions (as if Snorlax is actually traveling somewhere via hoping spawn points). I cant prove any of these other things yet but if the scanning app can register the coords for when the nearby  tracker updates and what it updates as, I think its possible to simply triangulate directly onto the spawn points.

**Comment by EternisRequiem** [comment]
@RvdHNL  Actually, if you look to see where that was FIRST posted, that was several days ago, before this change, and when they first implemented the 5s scan.  (Though I heard there was some work done since they were supporting two versions of the game simultaneously or something?  Maybe that's the case?)

**Comment by RvdHNL** [comment]
Found this:

SUMMARY
API stopped accepting requests from any sources which are not the actual client. The API needs a value "unknown 6", this value was already in the API in previous versions, but now the server is validating it. Only the actual client can create a valid "unknown6". We dont actually 100% know that it is indeed "unknown6" that is being validated, but it would make sense since its a big piece of data which isnt recreateable.
It is not as easy as locating where any updates made changes because the unknown6 was already being calculated and sent in previous versions but not validated by the server.
It doesnt really matter exactly what values go into the unknown6. CRACKING/bruteforcing the code is impossible because the key alone wouldnt do it. We need to get to the piece of code that makes "unknown6". The key and the way to calculate unknown6 is somewhere within the code and were trying to find it.
We are trying to locate where the app calculates unknown6 in order to be able to recreate out own valid unknown6's. If we do that we have a working API again.
This is hard because parts of the code are not easily accessible. We need people that can decompile and document parts of the code!
Please keep the talking only about the API, no personal lives, let the programmers do their job.
Again were not looking for theories on how unknown6 is created, what values go in it. We will 99% sure only get a working API if we locate and understand the code creating unknown6.
Latest so: http://www.speedyshare.com/F2F7J/libNianticLabsPlugin.so#

**Comment by blonkpuff** [comment]
@pill0ws 

You personally don't need to scan all over the world, no,  but think from the perspective of the app. It would have to scan wherever the current user is and wherever the current user travels and start building its guess on where pokemon always spawn. That would be something it has to work out over several scans of the area.  

Right now the app probably just takes in data from the pokego servers and displays it to you along with time remaining, if it's lured, etc.  But you're now asking it to store the location of every pokemon it finds in memory and compare that to the future spawns it finds to the database it's already compiled and further decide when they intersect enough that it becomes a defined spawn point. Then it has to adjust its scans to only hit those areas. That procedure would apply to everyone using that app and all I mean is when you consider adding all that functionality to an app that's currently just doing call/display and a bit of filtering...that's a lot to ask of the devs!

I haven't picked through the api myself to know what it's all delivering. I'm sure the devs could come up with a prediction algorithm given enough time if they knew enough. In such a case it's probable the that the clues you're putting together would very much help such an algorithm function isolate spawnpoints, (just like the Turing machine used "Hail Hitler" as a fixed key to decipher the rest of the axis power messages in WWII when they found out it was included at the end of every transmission!) 

But yeah, I guess I'm saying is it worth it for the devs to program when they aren't getting paid and when a complicated predict feature might not only take a ton of time but also might make itself more suspicious?

**Comment by pill0ws** [comment]
@blonkpuff 
Well I wasnt suggesting a predict feature, merely suggesting if orientation was available information that it could be valuable. It would only ever be available though once a pokemon had been revealed by the scanner. I certainly am not suggesting new features, just throwing out ideas from the observation I made playing with the scanner for hours. I am going to stop talking though, these guys seem to be hard at work and I dont want to create anymore noise. If anyone is interested though I could organize all the info I saved, it was just copypasta from the Apps feedback though but includes coords and despawn times for lots of things and a ton of Snorlax

**Comment by EternisRequiem** [comment]
So... why does allegedly https://map.goradar.io/ work?  I can't get it to change location for the life of me, however :P haha  But it's something people can use in the interim.  It's crowdsourced by people with the ios app apparently?

**Comment by Josp1337** [comment]
https://map.goradar.io/ is fake and is not accurate !

**Comment by EternisRequiem** [comment]
Several people on reddit stated that it was working for them (and it is because of the crowdsourced nature of it)

**Comment by EternisRequiem** [comment]
[Looks like someone got it?!](https://github.com/AeonLucid/POGOProtos/issues/131#issuecomment-237475735)

**Comment by blonkpuff** [comment]
@EternisRequiem 

It (allegedly) works because Niantic can't do anything to block that. As I understand it it works by people reporting pokemon they find to a 3rd party site, no?  That can't be regulated by their app. Trade off would be crappy reliability and no real sense for despawn times.

**Comment by EternisRequiem** [comment]
@blonkpuff It's people using PokemonGo++, so an edited app that I assume takes data they're seeing and then sends it off, so it ends up being reliable and with proper respawn timers, allegedly.  (This is what people keep reporting)

**Comment by blonkpuff** [comment]
^Ah interesting, I wonder if Niantic is onto that yet.   As you said though the location marker seems stuck?

**Comment by ShiroTora** [comment]
so did they fix it ? or is there still no app/prog that can track ?

**Comment by obamaphone** [comment]
Well, I am a 1337 h4x0r and I say FUCK NIANTIC!

**Comment by Sht12** [comment]
Noobie Question: Do i have to download a new version of the PoGo Map application whenever there is an update? or Is there just an update file..

**Comment by ShiroTora** [comment]
i know how you feel, i finay started going outside with my bike to catch pokemon arround me in a  radiius of 10km, right now im back locked up in my room thinking what to do , these scanners where rly helpfull, without a tracker it is worthless to spot rare pokemon unless you sit all day with a pokestop with lure, but thats same as sitting in my room in front of my laptop waiting for a solution,  so i can go outside again, i never was an outsider, i always played lots online games on laptop , right now this pokemon go stuff is going trough my head , i cant even focus in the online games i normaly play

**Comment by ShiroTora** [comment]
dont know if you readed, nianticencrypted the api, so all scannes/maps wont work for time being

**Comment by sushab** [comment]
@ ShiroTora
ues the game is totally boring like this. The radar apps, made the 'hardcore' scene interesting; knowing spawn times, spawn locations, getting there on time, while competing, in the real world. Now it just feels like the only ALLOWED way of playing is casual/noob/child play. 

Niantec servers around the world should have no problem taking the loads (maybe not the first week ok launch), it almost feels like NIANTEC is seriously expecting to have complete monopoly of the product and related/periferal services, which these days is highly unrealistic and seems to suggest that Niantec are more administrators than programmers which doesn't bode well for the future in terms of the so needed new game modes and new content.

**Comment by muntedfish** [comment]
What kills me is my pokedex is at 140. I just need a porygon and Chansey! I have seen several Chansey but miss took them for Clifable grr. Scanner has shown Chansey several times... just need Porygon then done after that! Now its just up to bling luck...

**Comment by ShiroTora** [comment]
well i am on 107/151 pokemon, i rly want to get more , also a question since i dont play that long yet, just a week, should i lvl the pokemon 1st before evolution or can i direct evolve them without any side effects ?

**Comment by lazyTrainer** [comment]
Where is the dev thread?  I'd like to follow it, not people comparing the size of their poke dicks.

**Comment by HuXu77** [comment]
@lazyTrainer https://github.com/AeonLucid/POGOProtos/issues/131

**Comment by pcecere** [comment]
@muntedfish  nobody here cares about your Porygon.

**Comment by starstremr** [comment]
what i don't understand why would niantic ask pokevision and others to stop there sites if there just going to do what they did with the mapping apps and programs?

**Comment by pill0ws** [comment]
Not sure I get that question, Are you asking why Niantic would want to remove a site that offers a service that they themself intend on offering? or are you asking why would they bother contacting Pokevision when they planned on breaking everything anyways? I'd say the fanswer to the first question is pretty obvious. The second question, because Pokevision was HUGE in no time at all. When and if workarounds for their security measures arise, it would easily explode onto the scene with a name that is already well known. By asking Pokevision to stop, they can be sure that the next time this happens they will only have to deal with a 10CP Squirtle and not a 3kCP Blastoise. Sure the Squirtle would evolve quickly, but the Blastoise is already a contender. It's easier to just revive the Blastoise than evolve another, they want that advantage

**Comment by starstremr** [comment]
ok gotcha makes sense. ty

**Comment by joshuacane** [comment]
I've been using Go Radar for the past few days and it seems reliable enough. My only issue is that it doesn't always show the pokémon that appear on my in app radar. The location of pokémon that do appear on Go Radar are pretty accurate in both time and location. I've managed to catch enough bulbasaurs for a venusaur today because of the app. As good as it is, it's still not as good as this app.

**Comment by Shogunz1** [comment]
Which app? I type in Go Radar and it gives me a million different ones lol.

**Comment by mchristopher** [comment]
I'm closing this as the latest fix addresses the issue.
